readme.md

```diff
@@ -0,0 +1,47 @@
+# developers-italia-api for the software catalog of Developers Italia
+API for the developers.italia.it public software collection
+
+## Table of Contents
+#### [Requirements](#requirements)
+#### [Description](#description)
+#### [Documentation](#documentation)
+
+# [Description](#description)
+
+Developers Italia API is a RESTful API that provides information about the catalog of Free and Open Source software aimed to Public Administrations.
+
+# [Requirements](#requirements)
+Developers Italia API requires the following:
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
```

Comment:
I'd focus more on the application requirements instead of the deployment / developer environment (ie. the API just needs PostgreSQL)